client/state/login/actions.js

```diff
@@ -434,7 +434,10 @@ export const logoutUser = ( redirectTo ) => ( dispatch, getState ) => {
 
 export const fetchOAuth2ClientData = ( clientId ) => dispatch => {
 	dispatch( {
-		type: OAUTH2_CLIENT_DATA_REQUEST
+		type: OAUTH2_CLIENT_DATA_REQUEST,
```

Comment:
> I used the same format as the client data stored in the `oauth2.clients` store. We could pass clientId there and have `[ OAUTH2_CLIENT_DATA_REQUEST ]: ( state, { clientId } ) => { id: clientId },` in the reducer, not sure which is better

I think `[ OAUTH2_CLIENT_DATA_REQUEST ]: ( state, { clientId } ) => { id: clientId },` is better than `[ OAUTH2_CLIENT_DATA_REQUEST ]: ( state, { data } ) => data` since it's more explicit.